resources/styles/components/scores/table.less

```diff
@@ -0,0 +1,472 @@
+        }
+      }
+    }
+  }
+  .name-cell {
+    .name {
+      text-align: left;
+      padding-left: 8px;
+    }
+  }
+
+
+
+  .filter-row {
+    width: 100%;
+    height: 70px;
+    display: block;
+    div {
+      float: left;
+    }
+    div + div {
+      margin-left: 10px;
+    }
+    .filter-group {
+      box-shadow: none;
+      button {
+        &.btn-default {
+          border-radius: 0px;
+        }
+        &.btn-primary {
+          border-radius: 0px;
+          border: @tutor-tertiary-light;
+          background: @tutor-tertiary-light;
+          height: 32px;
+          &:hover {
+            background: @tutor-tertiary-light;
+          }
+        }
+      }
+    }
+    .filter-item {
+      &:first-child {
+        width: 250px;
+      }
+      float: left;
+      display: flex;
+      align-items: center;
+      .filter-label {
+        font-size: 1.4rem;
+      }
+    }
+  }
+
+}
+
+.scores-scores-tooltip-completed-info {
+  @completed-popover-row-height: 30px;
+  height: auto;
+  .popover-content {
+    height: 100%;
+    overflow: hidden;
+    padding: 0;
+  }
+  .info {
+    display:table;
+    .row {
+      display: table-row;
+      &:nth-child(1) {
+        background: darken(@tutor-info-bg, 5%);
+      }
+      &:nth-child(2) {
+        color: @tutor-neutral;
+      }
+      &:nth-child(3) {
+        background: @tutor-neutral-lighter;
+        color: @tutor-black;
+      }
+      div {
+        font-weight: bold;
+        width: 150px;
+        height: @completed-popover-row-height;
+        vertical-align: middle;
+        display: table-cell;
+        padding-left: 20px;
+      }
+    }
+  }
+
+}
+
+.late-work-info-popover {
+  height: 98px;
+  min-width: 315px;
+  border-radius: 0;
+  border: 1px solid @tutor-neutral-light;
+  &.accepted {
+    .popover-title {
+      color: @tutor-black;
+      background: @tutor-info-bg;
+    }
+    .popover-content {
+      .status {
+        color: @pie-color-late !important;
+      }
+    }
+  }
+  .popover-title {
+    text-align: center;
+    font-weight: bold;
+    padding-top: 6px;
+    font-size: 1em;
+    height: 30px;
+    border-radius: 0;
+    color: @tutor-white;
+    background: @tutor-primary;
+  }
+  .popover-content {
+    padding: 3px;
+    .late-status {
+      height: 60px;
+      .description {
+        height: inherit;
+        display: table;
+        width: 48%;
+        float: left;
+        font-weight: bold;
+      }
+      
+      .title {
+        display: table-cell;
+        vertical-align: middle;
+        text-align: center;
+
+      }
+      .status {
+        display: table-cell;
+        vertical-align: middle;
+        color: @tutor-primary;
+        text-align: center;
+      }
+    }
+    .late-button {
+      padding-left: 2px !important;
+      padding-right: 2px !important;
+      width: 48%;
+      text-transform: none !important;
+      font-size: 1.1em !important;
+      float: right;
+    }
```

Comment:
completely personal preference on my part, but I like `flexbox` a lot more than good 'ole `table-cell` and friends.